lib/elixir/lib/option_parser.ex

```diff
@@ -100,7 +103,7 @@ defmodule OptionParser do
       assumes the type of `:switch_name` will be `:string`.
 
   To use `:keep` with a type other than `:string`, use a list as the type
-  for the switch. For example: `[foo: [:integer, :keep]]`.
+  for the switch. For example: `foo: [:integer, :keep]`.
```

Comment:
I would keep the list format from before especially because it is a nested list and you wouldn't be able to write the new version in any actual code sample.